routes/web.php

```diff
@@ -165,42 +161,25 @@
 
 Route::get('legal/{page}', 'LegalController@show')->name('legal');
 
-// ranking section
-Route::get('/rankings/{mode?}', function ($mode = 'osu') {
-    if (!array_key_exists($mode, App\Models\Beatmap::MODES)) {
-        abort(404);
-    }
-
-    return Redirect::route('rankings', ['mode' => $mode, 'type' => 'performance']);
-});
-Route::get('/rankings/{mode}/{type}', 'RankingController@index')->name('rankings');
+Route::get('rankings/{mode?}/{type?}', 'RankingController@index')->name('rankings');
 
 Route::post('session', 'SessionsController@store')->name('login');
 Route::delete('session', 'SessionsController@destroy')->name('logout');
 
 Route::post('users/check-username-availability', 'UsersController@checkUsernameAvailability')->name('users.check-username-availability');
 Route::post('users/check-username-exists', 'UsersController@checkUsernameExists')->name('users.check-username-exists');
 Route::get('users/disabled', 'UsersController@disabled')->name('users.disabled');
-Route::get('users/register', function () {
-    return Redirect::to('https://osu.ppy.sh/p/register');
-})->name('users.register');
 Route::get('users/{id}/card', 'UsersController@card')->name('users.card');
 Route::resource('users', 'UsersController', ['only' => ['show']]);
 
 Route::group(['prefix' => 'help'], function () {
     // help section
-    Route::get('wiki/{page}', 'WikiController@show')->name('wiki.show')->where('page', '.+');
+    Route::get('wiki/{page?}', 'WikiController@show')->name('wiki.show');
```

Comment:
woops, didn't think to test that case

I probably should have asked about that *before* deleting it :^)